include/cpr/util.h

```diff
@@ -42,6 +42,11 @@ std::string urlDecode(const std::string& s);
 void secureStringClear(std::string& s);
 bool isTrue(const std::string& s);
 
+/**
+ * Parse string timestamp to time_t
```

Comment:
```suggestion
 * Parses the given std::string into time_t (unix ms).
 * This parsing happens time_t size agnostic since time_t does not use the same underlying type on all systems/compilers.
```

Reply:
Very good annotation, I will modify it immediately.